service/src/test/java/uk/gov/hmcts/reform/fpl/service/email/content/cmo/DraftOrdersUploadedContentProviderTest.java

```diff
@@ -1,135 +0,0 @@
-package uk.gov.hmcts.reform.fpl.service.email.content.cmo;
-
-import org.junit.jupiter.api.Test;
-import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.test.context.ContextConfiguration;
-import uk.gov.hmcts.reform.fpl.enums.JudgeOrMagistrateTitle;
-import uk.gov.hmcts.reform.fpl.enums.TabUrlAnchor;
-import uk.gov.hmcts.reform.fpl.model.CaseData;
-import uk.gov.hmcts.reform.fpl.model.HearingBooking;
-import uk.gov.hmcts.reform.fpl.model.Respondent;
-import uk.gov.hmcts.reform.fpl.model.RespondentParty;
-import uk.gov.hmcts.reform.fpl.model.common.JudgeAndLegalAdvisor;
-import uk.gov.hmcts.reform.fpl.model.notify.cmo.DraftOrdersUploadedTemplate;
-import uk.gov.hmcts.reform.fpl.model.order.HearingOrder;
-import uk.gov.hmcts.reform.fpl.service.email.content.AbstractEmailContentProviderTest;
-
-import java.time.LocalDateTime;
-import java.time.Month;
-import java.util.List;
-import java.util.stream.Collectors;
-import java.util.stream.Stream;
-
-import static org.assertj.core.api.Assertions.assertThat;
-import static uk.gov.hmcts.reform.fpl.enums.HearingType.CASE_MANAGEMENT;
-import static uk.gov.hmcts.reform.fpl.enums.JudgeOrMagistrateTitle.HER_HONOUR_JUDGE;
-import static uk.gov.hmcts.reform.fpl.enums.JudgeOrMagistrateTitle.HIS_HONOUR_JUDGE;
-import static uk.gov.hmcts.reform.fpl.enums.JudgeOrMagistrateTitle.MAGISTRATES;
-import static uk.gov.hmcts.reform.fpl.utils.ElementUtils.wrapElements;
-
-@ContextConfiguration(classes = {DraftOrdersUploadedContentProvider.class})
```

Comment:
is the right test class to remove? DraftOrdersUploadedContentProvider vs DraftCMOUploadedContentProvider

Reply:
hm oops I think you are right 🤦